components/photoslider.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Box, Button, Container, HStack } from '@chakra-ui/react';
+
+function PhotoSlider() {
```

Comment:
change this line to `export default PhotoSlider() {` to clean up the code

also remove the last export default at the end

Reply:
replaced with slider